bin/trick-CP

```diff
@@ -135,14 +136,20 @@ convert_swig: build/S_library_swig
 S_define_exp:
 	$(TRICK_CC) -E -C -xc++ ${TRICK_SFLAGS} $(TRICK_SYSTEM_SFLAGS) S_define > $@
 
+# prints the value of a makefile variable, example invocation "make print-TRICK_CXXFLAGS"
+# This rule is used by trick-config
```

Comment:
Reading this comment reminded me that bin/trick-config uses Makefile.sim,  I would suggest creating a small makefile in the same directory that Makefile.sim lives in that includes Makefile.common and has one rule for "print-%:"